File: template/app/src/main/java/com/xmartlabs/template/ui/onboarding/OnboardingActivity.java
package com.xmartlabs.template.ui.onboarding;

import android.support.annotation.NonNull;

import com.f2prateek.dart.HensonNavigable;
import com.xmartlabs.bigbang.ui.BaseFragment;
import com.xmartlabs.bigbang.ui.SingleFragmentActivity;

@HensonNavigable
public class OnboardingActivity extends SingleFragmentActivity {
  @NonNull
  @Override
  protected BaseFragment createFragment() {
    return new OnboardingFragmentBuilder().build();
  }
}

File: template/app/src/main/java/com/xmartlabs/template/ui/onboarding/OnboardingActivity.kt
package com.xmartlabs.template.ui.onboarding

import com.f2prateek.dart.HensonNavigable
import com.xmartlabs.bigbang.ui.SingleFragmentActivity

@HensonNavigable
class OnboardingActivity : SingleFragmentActivity() {
  override fun createFragment() = OnboardingFragmentBuilder().build()
}
